cmake/SeracBasics.cmake

```diff
@@ -18,8 +18,6 @@ option(SERAC_ENABLE_CODEVELOP
        "Enable Serac's codevelop build (MFEM and Axom included as CMake subdirectories)"
        OFF)
 
-option(SERAC_ENABLE_LUMBERJACK "Enable Axom's Lumberjack component" ON)
```

Comment:
This was originally optional due to a bug (and really a lack of production level proving) in Lumberjack.  It is now more trusted and tested than the alternative in SLIC, so I am removing the option which noone ever exercised.